scripts/verify-detector-activation-evidence-handoff.sh

```
@@ -78,6 +107,35 @@ for phrase in "${required_manifest_text[@]}"; do
+  "Release identifier: aegisops-single-customer-pilot-2026-04-27-c4527e5"
+  "Candidate rule identifier: github-audit-privilege-change"
+  "Source-family package: GitHub audit via reviewed Wazuh-backed intake boundary"
+  "Lifecycle state: staging-review-accepted"
+  "Activation owner: pilot-detector-owner-redacted"
+  "Disable owner: pilot-disable-owner-redacted"
+  "Rollback owner: pilot-rollback-owner-redacted"
+  "Expected alert volume: 0-3 alerts per business day during the first pilot week"
+  "False-positive review: Routine repository administration, access review cleanup, automation identity behavior, and scheduled maintenance were reviewed as expected benign cases"
+  "Next-review date: 2026-05-04"
+  "Fixture evidence: control-plane/tests/fixtures/wazuh/github-audit-alert.json"
+  'Parser evidence: decoder.name `github_audit`, rule.id `github-audit-privilege-change`, and source-family parser coverage verified by the GitHub audit detector activation candidate tests'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)